.github/workflows/ci.yml

```diff
@@ -129,7 +132,7 @@ jobs:
       - name: Install python test dependencies
         run: |
           python -m pip install -U pip setuptools
-          pip install setuptools-rust pytest pytest-benchmark tox tox-venv
+          pip install setuptools-rust pytest pytest-benchmark tox
```

Comment:
I removed `tox-venv` because it caused issues with PyPy 3.7 on windows, and according to its repository [it's deprecated](https://github.com/tox-dev/tox-venv#tox-venv) anyway.